e2e_tests/conftest.py

```diff
@@ -33,3 +33,18 @@ async def token(verify) -> str:
 
         assert token is not None, "Token not returned"
         return token if (response.status_code == status.HTTP_200_OK) else None
+
+
+@pytest.fixture
+async def workspace_owner_token(verify) -> str:
+    async with AsyncClient(verify=verify) as client:
+
+        headers = {'Content-Type': "application/x-www-form-urlencoded"}
+        payload = f"grant_type=password&resource={config.AUTH_APP_CLIENT_ID}&username={config.USERNAME}&password={config.PASSWORD}&scope=open_id+profile+email&client_id={config.AUTH_APP_CLIENT_ID}"
+
+        url = f"https://login.microsoftonline.com/{config.AUTH_TENANT_ID}/oauth2/token"
+        response = await client.post(url, headers=headers, data=payload)
+        token = response.json()["access_token"]
```

Comment:
@LizaShak I'm getting "invalid key" for `access_token` here: https://github.com/microsoft/AzureTRE/runs/4301934706?check_suite_focus=true

Any ideas?

Reply:
Sorted, issue with `client_id` in above command - was set to the workspace app ID. Also app registrations needed scope adding and admin consent granting.